_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,284 @@
+            </label>
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label inline-label" for="try_and_see_02">
+                <input type="checkbox" class="input input-checkbox replace-checkbox" name="edx_goals_02" id="try_and_see_02" value="Try it out and see what I can learn">
+                <span class="input-label">Try it out and see what I can learn</span>
+            </label>
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label inline-label" for="new_job_02">
+                <input type="checkbox" class="input input-checkbox replace-checkbox" name="edx_goals_02" id="new_job_02" value="I need a new jobby job!" checked="checked">
+                <span class="input-label">I need a new jobby job!</span>
+            </label>
+        </div>
+    </fieldset>
+</div>
+
+<h4 class="hd-6">Textareas</h4>
+<div class="example-set">
+    <fieldset class="fieldset">
+        <legend class="group-legend is-hidden">Storytime!</legend>
+        <div class="field-wrapper">
+            <label class="form-label" for="textarea_01">Short textarea</label>
+            <textarea class="textarea input-textarea textarea-short" name="textarea_01" id="textarea_01"></textarea>
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label" for="textarea_02">Base textarea</label>
+            <textarea class="textarea input-textarea textarea-medium" name="textarea_02" id="textarea_02"></textarea>
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label" for="textarea_03">Tall textarea</label>
+            <textarea class="textarea input-textarea textarea-tall" name="textarea_03" id="textarea_03"></textarea>
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label" for="textarea_04">Block textarea</label>
+            <textarea class="textarea input-textarea textarea-medium textarea-block" name="textarea_04" id="textarea_04"></textarea>
+        </div>
+    </fieldset>
+</div>
+
+<h4 class="hd-6">Sliders</h4>
+<div class="example-set">
+    <div class="form-group">
+        <div class="field-wrapper">
+            <label class="form-label" for="slider_01">Slide it</label>
+            <input type="range" class="slider input-slider" name="slider_01" id="slider_01" min="1" max="12" step="1" value="6" role="slider" aria-min="1" aria-max="12" aria-now="6">
+        </div>
+    </div>
+</div>
+
+<h4 class="hd-6">Progress bars</h4>
+<div class="example-set">
+    <div class="form-group">
+        <div class="field-wrapper">
+            <label class="form-label" for="progress_01">You're doing great! Keep it up!</label>
+            <progress class="progress input-progress" name="progress_01" id="progress_01" role="progressbar" value="50" max="100" aria-valuenow="50" aria-valuemax="100"></progress>
+        </div>
+    </div>
+</div>
+
+<h4 class="hd-6">Example form</h4>
```

Comment:
Add ``.example-set-hd`` class in here for consistency and extra PL Doc site formatting.